### ISSUE TITLE
Horsecart Duplication

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLife#814]
I've seen him do it a few times, d3mon1cblack has managed to duplicate a [horsecart](https://www.twitch.tv/d3mon1cblack/clip/BumblingCreativeDogAMPEnergy-0qixh0W4oNE5_iVP?filter=clips&range=all&sort=time). From what I recall him saying, he thinks it has to do with how desert biomes force non-blacks to drop objects when player count is at least 30 players on a server.

SirStabbn managed to duplicate a horsecart once.

SoloAceMouse summarized demoni's recreation advice as follows:

"1) Must be non-desert expert - no black character
2) Horsecart must contain item[s]
3) Must start on non-desert tile and end on non-desert tile while crossing a snake in between"

Demoni's [picture](https://cdn.discordapp.com/attachments/328215300279500800/1109623651000655912/hold_ur_horses.png) 

Tarr reported duplicating horsecarts a few times.

**Comment by DougLefelhocz** [comment]
d3mon1cblack : "it might also have to do with klicking far away but im not that sure on that
cus it makes you glide the rest of the way"

Tarr: "If you click through a snake and don't click it should auto path you to the tile but you'll sort of slide instead of the normal horse walk"

**Comment by jasonrohrer** [comment]
Thank you for the detailed report!

**Comment by jasonrohrer** [comment]
Hmm... I can't reproduce this on my end.  I see the slide happen (when I click a long distance that crosses the desert and a snake).  But no duplication.

Can someone shoot a video of this happening?

**Comment by DougLefelhocz** [comment]
Tarr: "Took three tries and did it"

Liz: "three tries is actually a pretty low number, surprising"

**Comment by jasonrohrer** [comment]
Still waiting for a video showing this.

That would help me to reproduce it on my end (so I can see the exact configuration that makes it possible).

**Comment by TwistedHBG** [comment]
> Still waiting for a video showing this.
> 
> That would help me to reproduce it on my end (so I can see the exact configuration that makes it possible).

there's a video linked in the original bug report message

**Comment by jasonrohrer** [comment]
Whoops... didn't see that link!  Thank you.

Here's another one:

https://www.twitch.tv/trekkiea/clip/HilariousSmoothAppleAMPEnergyCherry-SaQoHLJoktE1X-Vo

**Comment by DougLefelhocz** [comment]
trekkiea: "I found the snake needs to be on a very specific tile for it to work. The only thing that seemed to cancel it were nearby objects that could disrupt where the horse or duped horse would be. Not all tiles worked. I think it had something to do with ne moving into non-desert tiles from snake+desert tile to work... buuut also needed to have more than one surrounding desert tile. Since single desert tiles wouldn't work
Also nearby players can disrupt it, since it moves snake and scares horse"

Trekkie's [picture](https://cdn.discordapp.com/attachments/328215300279500800/1113658068740345918/Screenshot_20230531_223920_Discord.jpg).

"X marks the sweet spots I found worked"
